### PR TITLE
Add documentation for creating a new addon

### PR DESCRIPTION
**Pull request by mehulkar**

### CODE REVIEW

File: guides/basic-use/cli-commands.md [hunk 1]
```diff
@@ -213,3 +213,27 @@ ember build --environment production
...
+
+### Example use
+
+This command below will create a folder called `x-select`.
+
+```shell
+ember addon ember-x-select
+```
+
+- To do: Document how to create addons that can be published in an `npm` scope/namespace
```
Comment (jenweber) [inline guides/basic-use/cli-commands.md]:
```suggestion
<!-- To do: Document how to create addons that can be published in an `npm` scope/namespace -->
```

File: guides/basic-use/cli-commands.md [hunk 2]
```diff
@@ -213,3 +213,27 @@ ember build --environment production
...
+
+### Format:
+
+```shell
+ember addon <my-addon-name> [options]
+```
+
+### What it does
+
+`ember addon` creates all the boilerplate files for a new Ember addon. It puts them in a folder with the same name as whatever we provide in place of `<my-app-name>`.
```
Comment (jenweber) [inline guides/basic-use/cli-commands.md]:
```suggestion
`ember addon` creates all the boilerplate files for a new Ember addon. It puts them in a folder with the same name as whatever we provide in place of `<my-addon-name>`.
```

File: guides/basic-use/cli-commands.md [hunk 3]
```diff
@@ -213,3 +213,27 @@ ember build --environment production
...
+
+### Example use
+
+This command below will create a folder called `x-select`.
+
+```shell
+ember addon ember-x-select
+```
+
```
Comment (jenweber) [inline guides/basic-use/cli-commands.md]:
```suggestion

### Learn more

- Follow the [tutorial](../../writing-addons/intro-tutorial/) to create your own Ember Addon!

```

File: guides/basic-use/cli-commands.md (hunk 2 above)
Comment (mehulkar) [reply guides/basic-use/cli-commands.md]: oops, good catch.

**Comment by mehulkar** [comment]
Tested the link locally and it looks like it's working! (Just ran the app with `ember server` and clicked on it. Not sure if that means it'll work in prod though

**Comment by mehulkar** [comment]
Netflify preview also looking good: https://deploy-preview-113--ember-cli-guides.netlify.com/release/basic-use/cli-commands/

File: guides/basic-use/cli-commands.md [hunk 4]
```diff
@@ -213,3 +213,31 @@ ember build --environment production
...
+ember addon <my-addon-name> [options]
+```
+
+### What it does
+
+`ember addon` creates all the boilerplate files for a new Ember addon. It puts them in a folder with the same name as whatever we provide in place of `<my-addon-name>`.
+
+### Example use
+
+This command below will create a folder called `x-select`.
```
Comment (jenweber) [inline guides/basic-use/cli-commands.md]:
```suggestion
This command below will create a folder called `ember-x-select`.
```

Comment (mehulkar) [reply guides/basic-use/cli-commands.md]: wow, that's just sloppy on my part. thanks for the proofread!

**Pull request merged**